JitsiParticipant.js

```diff
@@ -20,8 +20,9 @@ export default class JitsiParticipant {
      * @param displayName
      * @param {Boolean} hidden - True if the new JitsiParticipant instance is to
      * represent a hidden participant; otherwise, false.
+     * @param {string} starsId - optional participant statsId
      */
-    constructor(jid, conference, displayName, hidden) {
+    constructor(jid, conference, displayName, hidden, statsId) {
```

Comment:
Trivial nits: typo in "starsId" and maybe it should be "statsID" to be consistent with the capitalization in the config?

Reply:
Ok, will change it. Its confusing cause switching between project (javascript and java) and in one place is statsID and in the other statsID and in the colibri messages its stats-id. I should check all PRs :)